tests/events.test.js

```diff
@@ -0,0 +1,15 @@
+import events from '../src/events';
+
+test('contains an on property', () => {
+    expect(events).toHaveProperty("on");
+});
+
+test('contains an off property', () => {
+    expect(events).toHaveProperty("off");
+});
+
+test('contains a trigger property', () => {
+    expect(events).toHaveProperty("trigger");
+});
+
+
```

Comment:
The only issue I see is these trailing newlines, it's a pretty straightforward change otherwise.